Source_Code/Client/App/views/CardPanel.java

```diff
@@ -60,7 +61,10 @@ public CardPanel( Card card )
       try
       {
          root = new File(Thread.currentThread().getContextClassLoader().getResource("").toURI());
-         picture = ImageIO.read( new File( root, "/../Source_Code/Client/App/Resources/" + cardName + ".png") );
+
+         picture = ImageIO.read( new File( root, "../../../../Sqrt_-11_Clue-Less/Source_Code/Client/App/Resources/" + cardName + ".png") );

```

Comment:
Can't we use `System.getProperty("user.dir")` to make this a little more robust?